src/main/kotlin/uk/gov/dluhc/printapi/jobs/FinalRetentionPeriodDataRemovalJob.kt

```diff
@@ -5,15 +5,18 @@ import org.springframework.scheduling.annotation.Scheduled
 import org.springframework.stereotype.Component
 import uk.gov.dluhc.printapi.database.entity.SourceType.VOTER_CARD
 import uk.gov.dluhc.printapi.service.CertificateDataRetentionService
+import uk.gov.dluhc.printapi.service.TemporaryCertificateDataRetentionService
 
 @Component
 class FinalRetentionPeriodDataRemovalJob(
-    private val certificateDataRetentionService: CertificateDataRetentionService
+    private val certificateDataRetentionService: CertificateDataRetentionService,
+    private val temporaryCertificateDataRetentionService: TemporaryCertificateDataRetentionService
 ) {
 
     @Scheduled(cron = "\${jobs.remove-vca-final-retention-period-data.cron}")
     @SchedulerLock(name = "\${jobs.remove-vca-final-retention-period-data.name}")
     fun removeVoterCardFinalRetentionPeriodData() {
+        temporaryCertificateDataRetentionService.removeTemporaryCertificateData(sourceType = VOTER_CARD)
```

Comment:
I considered creating a separate scheduled method, but I think it's ok to remove the temporary cert data at the same time (both get removed on 1st July).

Reply:
Your Could use the same job, but may be control deleting of temp cert and aed data with 2 flags. 

Retain-temp-cert: true
Retain-aed-data : true

Reply:
I don't see a need if I'm honest. I can't see why we'd want to disable the removal of data (other than a feature flag, which we don't need in this instance)?

I originally made the number of days to retain data configurable, partly because counting in days gives you a lot of flexibility (you can set it to a high number to cover months and years if necessary), but mainly because the legislation and requirements were less clear, but I think it's different here. There's both process and legislation involved and if in the unlikely event it changed from say months to days then we have to make a code change anyway.